website/i18n/en.json

```diff
@@ -489,1339 +493,1339 @@
         "title": "Button reference",
         "sidebar_label": "Button reference"
       },
-      "version-12.0/version-12.0-about-the-consoles": {
+      "version-12.0/about-the-consoles": {
```

Comment:
I realise that these changes are auto generated but they shouldn't be checked in because it makes life quite annoying for our translators. Could this be edited so it only includes additions please?

Reply:
Do you mean "en.json" shouldn't be checked in at all? Or if a new section is added does it need to be updated?

Reply:
Just the new sections should be updated, so basically all of these renames from `"version-12.0/version-12.0-about-the-consoles"` to `"version-12.0/about-the-consoles"` should not be included

Reply:
OK I have edited it to only include the current version